projects/aca-content/src/lib/ui/variables/variables.scss

```diff
@@ -73,6 +77,13 @@ $defaults: (
   --theme-about-panel-border-color: $grey-background,
   --theme-about-panel-background-color: #ffffff,
   --theme-about-panel-title-color: #212121,
+  --theme-sidenav-background-color: $sidenav-background-color,
+  --theme-selected-text-color: $selected-text-color,
+  --theme-selected-background-color: $selected-background-color,
+  --theme-hover-background-color: $grey-text-background,
+  --theme-action-button-text-color: $action-button-text-color,
+  --theme-header-border-color: $grey-background,
+  --theme-tooltip-background-color: $pagination-background-color,
```

Comment:
this does not make sense, each time a pagination theme changed, it will be affecting the tooltips?